### ISSUE TITLE
upsert is not working

### ISSUE THREAD

**Issue by suryasr007** [ligyxy/DictMySQLdb#7]
I tried use upsert(). But nothing is reflecting in database, However it is not generating even errors
```
db.upsert(table = SERVER_STORAGE_TABLE,
                 value = {'name':server, 'ping':'no'}
                  update_columns=['ping'])
```
Here is my code

**Comment by gyli** [comment]
This query looks good to me, but it also depends on the data and db structure, like if there is UNIQUE index or not.

So I think you can set db.debug = True and it will print the query it should be running, then you can debug the raw sql query directly.

**Comment by gyli** [comment]
Thx, this is fixed in 0.6.6
